lib/include/event-driven/vDrawSkin.h

```diff
@@ -0,0 +1,1159 @@
+                    yoffset=handL.yoffset;
+                    scaling =handL.scaling;
+                    radius = handL.radius;
+                    noise = handL.noise;
+                    max_value = handL.max_value;
+                }
+                else {
+                    yWarning("Unable to build the taxel map from file");
+                    pos = default_pos;
+                }
+            }
+
+        };
+
+};
+
+class isoDrawSkin : public isoDraw {
+
+    private:
+
+    loadMap tmap;
+
+    int radius;
+
+    public:
+
+    void initialise(){
+
+
+        Xlimit = 800;
+        Ylimit = 800;
+
+        isoDraw :: initialise();
+        tmap.initialise("leftHand");
+
+        radius = tmap.radius-1;
+
+    };
+
+    static const std::string drawtype;
+    virtual void draw(cv::Mat &image, const ev::vQueue &eSet, int vTime);
+    virtual std::string getDrawType();
+    virtual std::string getEventType();
+};
+
+
+
+
+class skinDraw : public vDraw {
+
+private:
+
+    loadMap tmap;
+    cv::Mat baseimage;
+
+    int radius;
+
+public:
+
+    void initialise(){
+
+        Xlimit = 800;
+        Ylimit = 800;
+
+        baseimage = cv::Mat(cv::Size(Xlimit, Ylimit), CV_8UC3);
+        baseimage.setTo(255);
+        tmap.initialise("leftHand");
+        radius = tmap.radius;
+
+        int x;
+        int y;
+
+        yInfo("scaling = %d", tmap.scaling );
+        yInfo("xoffset = %d", tmap.xoffset );
+        yInfo("yoffset =%d", tmap.yoffset );
+
+        for (auto it = tmap.pos.begin(); it != tmap.pos.end(); it++)
+        {
+            x = tmap.xoffset + tmap.scaling* std :: get<0>(it->second);
+            y =  tmap.yoffset +tmap.scaling* std :: get<1>(it->second);
+            cv::Point centr_all(x, y);
+            cv::circle(baseimage, centr_all, radius, black,1, CV_AA);
+        }
+
+        };
+
+
+
+    virtual void resetImage(cv::Mat &image)
+    {
+        baseimage.copyTo(image);
+    }
+    static const std::string drawtype;
+    virtual void draw(cv::Mat &image, const ev::vQueue &eSet, int vTime);
+    virtual std::string getDrawType();
+    virtual std::string getEventType();
+
+};
+
+class skinsampleDraw : public vDraw {
+
+
+private:
+
+    loadMap tmap;
+    cv::Mat baseimage;
+    int radius,noise,max_value;
+
+public:
+  void initialise(){
+
+        Xlimit = 800;
+        Ylimit = 800;
+
+        baseimage = cv::Mat(cv::Size(Xlimit, Ylimit), CV_8UC3);
+        baseimage.setTo(255);
+
+        tmap.initialise("leftHand");
+
+        radius = tmap.radius;
+        noise = tmap.noise;
+        max_value=tmap.max_value;
+
+        int x;
+        int y;
+
+
+        for (auto it = tmap.pos.begin(); it != tmap.pos.end(); it++)
+        {
+            x = tmap.xoffset + tmap.scaling* std :: get<0>(it->second);
+            y =  tmap.yoffset +tmap.scaling* std :: get<1>(it->second);
+            cv::Point centr_all(x, y);
+            cv::circle(baseimage, centr_all, radius, black,1, CV_AA);
```

Comment:
This is the only part that really changes from the previous implementations of this class